requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 -r requirements.txt
-codecov==2.0.9
+codecov
```

Comment:
Why do you remove the specific versions? I had problems with running the tests since I had a too old version of PyAML.

Reply:
ok, this was probably a mistake. PyCharm was complaining a lot about old dependencies but I think I also had some problem with incompatible dependencies when creating the docker image and taking away the versions fixed that. I'll go over it again and see if I can fix it.